doc/README.md

```diff
@@ -39,11 +39,13 @@
             * [.bytesRead](#BlockMap.ReadStream+bytesRead) : <code>Number</code>
             * [.position](#BlockMap.ReadStream+position) : <code>Number</code>
         * [.parse](#BlockMap.parse) ⇒ <code>[BlockMap](#BlockMap)</code>
+        * [.stringify](#BlockMap.stringify) ⇒ <code>String</code>
         * [.versions](#BlockMap.versions) : <code>Array</code>
         * [.create(options)](#BlockMap.create) ⇒ <code>[BlockMap](#BlockMap)</code>
         * [.fromJSON(data)](#BlockMap.fromJSON) ⇒ <code>[BlockMap](#BlockMap)</code>
         * [.createReadStream(filename, blockMap, options)](#BlockMap.createReadStream) ⇒ <code>[ReadStream](#BlockMap.ReadStream)</code>
         * [.parse(value, blockMap)](#BlockMap.parse) ⇒ <code>[BlockMap](#BlockMap)</code>
+        * [.stringify(blockMap, [options])](#BlockMap.stringify) ⇒ <code>String</code>
```

Comment:
Good catch! Somehow `jsdoc2md` doesn't resolve those; fixed in https://github.com/resin-io-modules/blockmap/pull/17